release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -443,6 +451,11 @@ In the following tables, features are marked with the following statuses:
 |Deprecated
 |Deprecated
 
+|iptables
+|Not applicable
+|Not applicable
+|Deprecated
```

Comment:
It's been deprecated since we moved to RHEL9 (4.13?). We're just trying to do a better job of _warning_ users about this now.